openage/__main__.py

```diff
@@ -137,7 +137,10 @@ def main(argv=None):
     args = cli.parse_args(argv)
 
     if sys.platform == 'win32':
-        add_dll_search_paths(args.dll_paths)
+        root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+        
+        if os.path.isdir(os.path.join(root, 'dll')):
+            add_dll_search_paths([os.path.join(root, 'dll')])
```

Comment:
why is `args.dll_paths` not used anymore?

If possible, problems with the parameter should be fixed instead of using a dirty workaround.

Reply:
I forgot to re added and I made this to be portable to be able to run without arg by default

Reply:
you should also add comments so it's clear what the code is doing and why